src/filesystem_fatfs.c

```diff
@@ -92,7 +92,7 @@ static int get_errno(FRESULT result) {
     case FR_MKFS_ABORTED:
         return EIO;
     case FR_TIMEOUT:
-        return EIO;
+        return EAGAIN;
```

Comment:
:+1:

Reply:
Yes, it is, but `fread`/`fwrite` is not required to set `errno` according to the POSIX standard anyway :wink: